src/components/buttons/BJbutton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+type BJButton = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  size: 'sm' | 'md' | 'lg';
+  btnName:string;
+};
+
+const BJButton = ({size,btnName,...props}: BJButton ) => {
+  const { className, ...restProps } = props;
+  const baseStyle = `${className} bg-green-400 hover:bg-blue-700`;
+
+  let sizeStyle = 'sm';
+  switch(size) {
+    case 'sm':
+      sizeStyle = 'py-1 px-2 text-sm';
+      break;
+    case 'md':
+      sizeStyle = 'py-2 px-4 text-base';
+      break;
+    case 'lg':
+      sizeStyle = 'py-3 px-6 text-lg';
+      break;
+    default:
+      break;
+  }
+  return (
+    <button {...restProps} className={`${baseStyle} ${sizeStyle}`}>
+        {btnName}
+    </button>
+  );
```

Comment:
`size`가 기본값이 주어진 상태면 이렇게도 쓸 수 있을 것 같네요..!

```suggestion
const sizeConfig = {
  sm: "py-1 px-2 text-sm",
  md: "py-2 px-4 text-base",
  lg: "py-3 px-6 text-lg",
};

return (
  <button {...restProps} className={`${baseStyle} ${sizeConfig[size]}`}>
    {btnName}
  </button>
);
```

Reply:
팀원분들께서 리뷰를 잘 달아주셔서 제가 따로 달아 드릴 리뷰가 크게 없는 것 같습니다...ㅎㅎ 
switch문을 사용하는 방법보다 종길님께서 달아주신 객체를 이용한 방법이 코드의 양을 조금 더 
줄일 수 있고 깔끔해보이는 것 같습니다 👍🏻